app/src/features/projects/components/ProjectPermitForm.tsx

```diff
@@ -56,63 +57,66 @@ const ProjectPermitForm: React.FC = () => {
-                          error={samplingConductedMeta.touched && Boolean(samplingConductedMeta.error)}
-                          displayEmpty
-                          inputProps={{ 'aria-label': 'Investment Category' }}>
-                          <MenuItem key={1} value="false">
-                            No
-                          </MenuItem>
-                          <MenuItem key={2} value="true">
-                            Yes
-                          </MenuItem>
-                        </Select>
-                        <FormHelperText>{samplingConductedMeta.error}</FormHelperText>
-                      </FormControl>
-                    </Grid>
-                    <Grid item xs={12} md={2}>
-                      <IconButton aria-label="delete" onClick={() => arrayHelpers.remove(index)}>
-                        <Delete />
-                      </IconButton>
-                    </Grid>
+                          error={permitNumberMeta.touched && Boolean(permitNumberMeta.error)}
+                          helperText={permitNumberMeta.error}
+                        />
+                      </Box>
+                      <Box flexBasis="30%" pl={1}>
+                        <FormControl variant="outlined" style={{ width: '100%' }}>
+                          <InputLabel id="sampling_conducted">Sampling Conducted</InputLabel>
+                          <Select
+                            id={`permits.[${index}].sampling_conducted`}
+                            name={`permits.[${index}].sampling_conducted`}
+                            labelId="sampling_conducted"
+                            label="Sampling Conducted"
+                            value={permit.sampling_conducted}
+                            onChange={handleChange}
+                            error={samplingConductedMeta.touched && Boolean(samplingConductedMeta.error)}
+                            displayEmpty
+                            inputProps={{ 'aria-label': 'Investment Category' }}>
+                            <MenuItem key={1} value="false">
+                              No
+                            </MenuItem>
+                            <MenuItem key={2} value="true">
+                              Yes
+                            </MenuItem>
+                          </Select>
+                          <FormHelperText>{samplingConductedMeta.error}</FormHelperText>
+                        </FormControl>
+                      </Box>
+                      <Box pt={0.5} pl={1}>
+                        <IconButton color="primary" aria-label="delete" onClick={() => arrayHelpers.remove(index)}>
+                          <Icon path={mdiTrashCanOutline} size={1} />
+                        </IconButton>
+                      </Box>
+                    </Box>
                   </Grid>
                 );
               })}
-            </Box>
-            <Box>
+            </Grid>
+            <Box pt={2}>
```

Comment:
Just curious: is there a reason why you would/wouldn't use padding vs margin here?